### ISSUE TITLE
ipmi_exporter - use maintainers' image

### ISSUE THREAD

**Issue by holmesb** [pnnl-miscscripts/miscscripts#57]
Hi,

Probably didn't exist when the chart was created, but Prometheus Community now provide a [docker image](https://hub.docker.com/r/prometheuscommunity/ipmi-exporter/) each time they release. Be good to switch to these, also because the image the chart uses is quite old now.

Cheers.

**Comment by kfox1111** [comment]
Yeah. looks like a pretty new image. yeah, we should switch to that. We could probably consider contributing the chart upstream too if anyone wants it.
